File: plugins/login-plugin/ios/InPlayerPluginConsts.swift
//
//  InPlayerPluginConsts.swift
//  InPlayerLogin
//
//  Created by Anton Kononenko on 4/2/20.
//

import Foundation
import InPlayerSDK
import ZappCore

struct InPlayerRejection {
    let code: String
    let message: String
}

let noCredentialsError = InPlayerRejection(code: "-1",
                                           message: "Can not initialize inPlayer, Payload does not contain expected inPlayer credentials: in_player_client_id, in_player_referrer")

let noExpectedPayloadParams = InPlayerRejection(code: "-2",
                                                message: "One of the expected params was not provided in Payload:")

File: plugins/login-plugin/android/src/main/java/com/applicaster/quickbrickinplayer/utils/InPlayerPluginConsts.kt
package com.applicaster.quickbrickinplayer.utils

class InPlayerRejection(code: String, message: String) {
    val code = code
    val message = message
}

internal val noCredentialsError = InPlayerRejection("-1",
        "Can not initialize inPlayer, Payload does not contain expected inPlayer credentials: in_player_client_id, in_player_referrer")

internal val noExpectedPayloadParams = InPlayerRejection("-2",
        "One of the expected params was not provided in Payload:")
